### ISSUE TITLE
External screen (HDMI) not working.

### ISSUE THREAD

**Issue by huotarih** [profzei/Matebook-X-Pro-2018#63]
Hey, I just upgraded to big sur beta 10, and everything else seems to be running smoothly, but my external screens aren't being recognized. I haven't used them in a while, propably last time was with catalina, and also different screens. But now I've tested with 2 screens and different hdmi cables and usb c adapters, and nothing works.

Anyone else with this issue?
Any ideas on how to get them working?

**Comment by profzei** [comment]
Hi, atm "nothing is working" if you use [BigSur_OC_EFI_POST.zip](https://github.com/profzei/Matebook-X-Pro-2018/blob/master/Wiki/BigSur_OC_EFI_POST.zip) file: this is **intentional** since Big Sur is still in **beta phase** and I'm interested only on minimal support for debugging hardware... 
You just need to update your `EFI/OC` folder with latest release... **since many posts in forums report crashes for macOS Big Sur Beta 10 and OpenCore 0.6.2** I suggest, instead, to update **only** your `config.plist` with the following code: 
```
                        <key>PciRoot(0x0)/Pci(0x2,0x0)</key>
			<dict>
				<key>AAPL,ig-platform-id</key>
				<data>BQDAhw==</data>
				<key>AAPL,slot-name</key>
				<string>Internal</string>
				<key>device-id</key>
				<data>wIcABQ==</data>
				<key>device_type</key>
				<string>VGA compatible controller</string>
				<key>disable-external-gpu</key>
				<data>AQAAAA==</data>
				<key>dpcd-max-link-rate</key>
				<data>FAAAAA==</data>
				<key>enable-dpcd-max-link-rate-fix</key>
				<data>AQAAAA==</data>
				<key>enable-hdmi20</key>
				<data>AQAAAA==</data>
				<key>enable-lspcon-support</key>
				<data>AQAAAA==</data>
				<key>force-online</key>
				<data>AQAAAA==</data>
				<key>framebuffer-con0-enable</key>
				<data>AQAAAA==</data>
				<key>framebuffer-con0-flags</key>
				<data>mAAAAA==</data>
				<key>framebuffer-con0-type</key>
				<data>AgAAAA==</data>
				<key>framebuffer-con1-enable</key>
				<data>AQAAAA==</data>
				<key>framebuffer-con1-flags</key>
				<data>xwMAAA==</data>
				<key>framebuffer-con1-has-lspcon</key>
				<data>AQAAAA==</data>
				<key>framebuffer-con1-preferred-lspcon-mode</key>
				<data>AQAAAA==</data>
				<key>framebuffer-con1-type</key>
				<data>AAgAAA==</data>
				<key>framebuffer-con2-enable</key>
				<data>AQAAAA==</data>
				<key>framebuffer-con2-flags</key>
				<data>xwMAAA==</data>
				<key>framebuffer-con2-has-lspcon</key>
				<data>AQAAAA==</data>
				<key>framebuffer-con2-preferred-lspcon-mode</key>
				<data>AQAAAA==</data>
				<key>framebuffer-con2-type</key>
				<data>AAgAAA==</data>
				<key>framebuffer-patch-enable</key>
				<data>AQAAAA==</data>
				<key>framebuffer-unifiedmem</key>
				<data>AAAAgA==</data>
				<key>hda-gfx</key>
				<string>onboard-1</string>
				<key>model</key>
				<string>Intel(R) UHD Graphics 620</string>
			</dict>
```

**Comment by huotarih** [comment]
This didn't work with the latest (1.6.0) version. Updated config.plist to your suggestions, but didn't help. If there's a solution that'd be great. Maybe I should downgrade? If you have any thoughts, lmk.

**Comment by huotarih** [comment]
Is this supposed to be fixed in 1.7.0 & Big Sur?
